ccmlib/scylla_repository.py

```diff
@@ -238,12 +238,20 @@ def setup(version, verbose=True, skip_downloads=False):
               - release:4.2.1
               - release:2020.1
               - release:2020.1.5
+            3. Debug versions of the above
+              - unstable/master:latest:debug
+              - unstable/enterprise:2020-08-18T14:49:18Z:debug
+              - release:4.3:debug
+              - release:4.2.1:debug
 
     :param verbose: if True, print progress during download
     :param skip_downloads: if True, skips the actual download of files for testing purposes
 
     """
     s3_url = ''
+    scylla_debug = version.split(':')[-1] == "debug"
```

Comment:
how about `version.endswith(":debug")`?
It seems clearer to me, especially when it is truncated using `version[:-6]`